src/ui/state/reporter.ts

```diff
@@ -0,0 +1,11 @@
+import { ExecutionPoint } from "@replayio/protocol";
+
+export interface Annotation {
```

Comment:
:hand: The early codebase defined TS types separately, but it's better to define the slice state types _inside_ the reducer/slice file.  Move this over to be alongside the corresponding reducer.

(Also, ideally our `UIState` type would _completely_ infer the entire type of the state from our root reducer, but we're still manually adding the slice state types to `UIState` atm.)